ceno_zkvm/src/tables/ops.rs

```diff
@@ -87,7 +87,49 @@ impl OpsTable for PowTable {
     }
 
     fn content() -> Vec<[u64; 3]> {
-        (0..Self::len() as u64).map(|b| [2, b, 1 << b]).collect()
+        (0..Self::len() as u64)
+            .map(|exponent| [2, exponent, 1 << exponent])
+            .collect()
+    }
+
+    fn pack(base: u64, exponent: u64) -> u64 {
+        assert_eq!(base, 2);
+        exponent
+    }
+
+    fn unpack(exponent: u64) -> (u64, u64) {
+        (2, exponent)
     }
 }
 pub type PowTableCircuit<E> = OpsTableCircuit<E, PowTable>;
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use crate::{
+        circuit_builder::{CircuitBuilder, ConstraintSystem},
+        tables::TableCircuit,
+    };
+    use goldilocks::{GoldilocksExt2 as E, SmallField};
+
+    #[test]
+    fn test_ops_table_assign() {
```

Comment:
one nitpick: I think this test only for pow_table, so probably `test_ops_pow_table_assign` more precisely